<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentDialog</class>
 <widget class="QDialog" name="StudentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: white;
}
QLineEdit{	
	border: 1px solid gray;
	border-radius: 6px;
	padding-left: 15px;
	height: 35px;
}
QDateEdit{
	border: 1px solid gray;
	border-radius: 6px;
	padding-left: 15px;
	height: 31px;
}
QComboBox{
	border: 2px solid white;
	border-radius: 8px;
	padding: 1px 18px 1px 3px;
	background-color: black;
	color: white;
	height: 35px;
	padding-left: 15px;
	font-weight: bold;
	selection-background-color:#5ce9ff;
}</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>60</y>
     <width>591</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add New Student</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>511</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Full Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="name_lineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Select Gender</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="gender_comboBox">
          <item>
           <property name="text">
            <string>Gender</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Male</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Female</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Other</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Select Class</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="class_comboBox">
          <item>
           <property name="text">
            <string>Grade</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Level 100</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Level 200</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Level 300</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Level 400</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Date of Birth</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dob_dateEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Department</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="department_comboBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>Select Department</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Computer Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Biomedical Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Materials Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Argricultural Engineering</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Food Processing Engineering</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Student ID</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="phone_lineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Phone Number</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="email_lineEdit">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>511</width>
     <height>45</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="saveStudent_btn_2">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>41</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{
	background-color: #ff3030;
	color: white;
	border: none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 15px;
}</string>
      </property>
      <property name="text">
       <string>Fingerprint</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>98</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="saveStudent_btn">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>41</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: #5ce9ff;
	color: white;
	border: none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 15px
}</string>
        </property>
        <property name="text">
         <string>Add Student</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel_btn">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>41</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	background-color: #585858;
	color: white;
	border: none;
	border-radius: 8px;
	font-weight: bold;
	font-size: 15px
}</string>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
